src/Donation/TransactionSubscriber.php

```diff
@@ -6,36 +6,43 @@
 use AppBundle\Mailer\MailerService;
 use AppBundle\Mailer\Message\DonationMessage;
 use Doctrine\Common\Persistence\ObjectManager;
+use Lexik\Bundle\PayboxBundle\Event\PayboxEvents;
 use Lexik\Bundle\PayboxBundle\Event\PayboxResponseEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
 
-class TransactionSuccessListener
+class TransactionSubscriber implements EventSubscriberInterface
 {
     private $manager;
     private $mailer;
     private $requestStack;
 
-    public function __construct(ObjectManager $manager, MailerService $mailer, RequestStack $requestStack)
+    public function __construct(ObjectManager $manager, MailerService $mailerTransactional, RequestStack $requestStack)
     {
         $this->manager = $manager;
-        $this->mailer = $mailer;
+        $this->mailer = $mailerTransactional;
```

Comment:
~~`$mailerTransactional`~~ -> `$mailer`

Reply:
I bound `$mailerTransactional` to the service `app.mailer.transactional`, he is used in many other services, if in long term we want use auto config/wire , I think it's a good idea to use bind arguments no ?